helm/prometheus-operator-app/values.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+grafana:
```

Comment:
Can you ensure the dashboards are still installed from the kube-prometheus-stack?

Reply:
Outdated due to recent changes